packages/datagateway-dataview/cypress/integration/table/isis/investigations.spec.ts

```diff
@@ -24,7 +24,8 @@ describe('ISIS - Investigations Table', () => {
     cy.get('[aria-rowcount="75"]').should('exist');
   });
 
-  it('should be able to resize a column', () => {
+  // TODO: Removal of the "visitID" column is breaking this resizing test.
+  it.skip('should be able to resize a column', () => {
```

Comment:
Another interesting part - dividing (windowWidth - 40 - 40) by 7 instead of 8 (which is correct, as there are now 7 columns instead of 8) gives 131.4285 while it detects the width of the columns to be 131.4166. Ridiculously close to correct but it makes you wonder why. But I'm also unsure where the values of 40 are coming from in that calculation.

Reply:
I was discussing this with @louise-davies and have yet to see why this isn't correct

Reply:
This has been solved by rounding the widths.